### ISSUE TITLE
CLI command to generate cache (mikro-orm cache:generate).

### ISSUE THREAD

**Issue by localnet** [mikro-orm/mikro-orm#225]
**Is your feature request related to a problem? Please describe.**
Deploy pre-built cache.

**Describe the solution you'd like**
CLI command to generate cache (`mikro-orm cache:generate`).

**Describe alternatives you've considered**
Generate the cache by running the compiled code locally.

**Additional context**
Now the cache is based in hash and it can be deployed. I think can be util to have the option of generate it from CLI.

For example, using shelljs with a `makefile`:
```javascript
target.build = function() {
  rm('-rf', 'dist/*');
  exec('tsc -p tsconfig.json');
  exec('mikro-orm cache:clear');
  exec('mikro-orm cache:generate');
}
```

**Comment by B4nan** [comment]
Agreed, was planning to add such cmd too

**Comment by B4nan** [comment]
Available in `3.0.0-alpha.34`
